test/store.spec.js

```diff
@@ -53,6 +53,11 @@ describe('JsonApiDataStore', () => {
         expect(article.title).to.eq('Cool article');
         expect(article.author).to.eq('Lucas');
       });
+
+      it('should not duplicate the attributes if the record is processed again', () => {
+        var article = store.sync(payload);
```

Comment:
Would you mind explicitly `sync`ing the payload twice here? There is no guarantee the tests will always be ran in the same order.